database/migrations/2020_08_20_141137_create_products_table.php

```diff
@@ -19,6 +19,7 @@ public function up()
             $table->string('name');
             $table->decimal('price', 5, 2);
             $table->text('description');
+            $table->unsignedBigInteger('brand_id');
```

Comment:
aggiungi una nuova riga
```suggestion
                        $table->foreign('brand_id')->references('id')->on('brands)->onDelete('cascade');
```